ansible/ci/group_vars/all.yml

```diff
@@ -2,7 +2,7 @@
 
 gcp_project: "stackrox-ci"
 gcp_auth_kind: "serviceaccount"
-gcp_service_account_file: /tmp/secret/stackrox-collector-e2e-tests/GOOGLE_CREDENTIALS_COLLECTOR_SVC_ACCT
+gcp_service_account_file: /tmp/secret/gcp-account.json
```

Comment:
Could this change interfere with OSCI?

Reply:
For now I'll copy the creds to the same place as OSCI would, to maintain compatibility